lib/reline/line_editor.rb

```diff
@@ -881,9 +883,10 @@ def editing_mode
         @completion_state = CompletionState::MENU
         perform_completion(list, true) if @config.show_all_if_ambiguous
       end
-      if not just_show_list and target < completed
-        @buffer_of_lines[@line_index] = (preposing + completed + completion_append_character.to_s + postposing).split("\n")[@line_index] || String.new(encoding: @encoding)
-        line_to_pointer = (preposing + completed + completion_append_character.to_s).split("\n")[@line_index] || String.new(encoding: @encoding)
+      unless just_show_list
+        _completion_append_character = continuable ? '' : completion_append_character.to_s
```

Comment:
underscore prefixed local variable represents unused variable in some rubocop rule.
How about just `append_character` or `completed += completion_append_character.to_s unless continuable`